tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -92,6 +94,13 @@ public Int32 SelectedRad
         private readonly GsDumps _availableGsDumps;
         private readonly GsDlls _availableGsDlls;
 
+        private List<FileSystemWatcher> _dllWatcher;
```

Comment:
I would dispose there on form dispose as you did on each reload.

Reply:
Done!